misc/io.github.grangerhub.Tremulous.appdata.xml

```diff
@@ -7,8 +7,8 @@
   <provides>
     <id>tremulous.desktop</id>
   </provides>
-  <launchable type="desktop-id">io.github.grangerhub.Tremulous.desktop</launchable>
-  <name>Tremulous</name>
+  <launchable type="desktop-id">Tremulous-Grangerhub.desktop</launchable>
+  <name>Tremulous-Grangerhub</name>
```

Comment:
Why? This will only make it confusing for our users. You can instead mention in the description that this is a "Grangerhub version of Tremulous". And it will add me much work to do to adjust the Flatpak to this unnecessary change. :-/